moped-editor/src/views/projects/projectView/ProjectComponents/mapStyleSettings.js

```diff
@@ -14,7 +14,8 @@ export const COLORS = {
   blueLight: "#a1cdf7",
   steel: "#607d8f",
   white: "#fff",
-  test: "#f28100",
+  orange: "#ffb300",
```

Comment:
Just in case you needed the hex code for UT's burnt orange, it's [#BF5700](https://brand.utexas.edu/identity/color/), you know, just in case. 😄